sorc/link_fv3gfs.sh

```diff
@@ -82,10 +84,10 @@ else
   done
 fi
 
-if [ -d ${pwd}/ufs_utils.fd ]; then
-  cd ${pwd}/ufs_utils.fd/sorc
-  ./link_fixdirs.sh $RUN_ENVIR $machine
-fi
+#if [ -d ${pwd}/ufs_utils.fd ]; then
```

Comment:
Why is this commented out?

Reply:
This is for debug purpose. I am restoring it now.